kythe/docs/schema/modules.txt

```diff
@@ -0,0 +1,148 @@
+// Copyright 2020 The Kythe Authors. All rights reserved.
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//   http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+Providing cross-references for module systems
+=============================================
+:Revision: 1.0
+:toc2:
+:toclevels: 3
+:priority: 999
+
+Many languages provide top-level mechanisms for organization that make
+writing large systems easier. These have explicit syntactic forms for
+expressing which components of a *module* are to be *exported* and which
+components are to be *imported* from other modules. It may also be possible
+to refer to the module itself as a first-class value in a program.
+
+A program using these features might look something like:
+
+[source,javascript]
```

Comment:
There's another kind of import/export possible in JS, a [default import](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Statements/import#Importing_defaults). You'd use it as `import foo from './module'`. The local identifier for the default export is arbitrary. Internally, default imports are the only way to import some kinds of things, and in those cases it's definitely preferable for the usages of `foo` to have a ref back to the original default-exported entity (or e.g. a protobuf message if it was generated, etc.).

Reply:
PTAL at the newest push.